cypress/utils/plugins/dashboards-flow-framework/commands.js

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  FF_FIXTURE_BASE_PATH,
+  INGEST_NODE_API_PATH,
+  SEARCH_NODE_API_PATH,
+} from '../../../utils/constants';
+
+Cypress.Commands.add('getElementByDataTestId', (testId) => {
+  return cy.get(`[data-testid="${testId}"]`);
+});
```

Comment:
nit: can you add this into the base commands file? Somewhere near https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/cypress/utils/commands.js#L268

This is to prevent duplicate commands written in different plugin files, for common helper commands like this one